typescript-operators/main.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-unused-vars */
+
+interface Customer {
+  name: string;
+  details?: {
+    address?: {
+      city: string;
+      street: string;
+      zip: string;
+    };
+    age?: number;
+  };
+}
+
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+const v5: Customer = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+
+// && Logical AND operator
+const and1 = v1 && 'v1 is truthy';
+console.log(and1);
+
```

Comment:
You're missing your `v2` check. Also you _only_ want to be running your `console.log` if `v1` is truthy here. Right now you are checking if it's is truthy, assigning a value to a variable, and logging the variable every time.